roofit/roofitcore/src/RooWorkspace.cxx

```diff
@@ -647,26 +626,18 @@ Bool_t RooWorkspace::import(const RooAbsArg& inArg,
     return kTRUE ;
   }
 
-  // Print a message for each imported node
-  iter = cloneSet2->createIterator() ;
-
   // Perform any auxiliary imports at this point
-  RooAbsArg* node ;
-  while((node=(RooAbsArg*)iter->Next())) {
+  for (const auto node : *cloneSet2) {
     if (node->importWorkspaceHook(*this)) {
       coutE(ObjectHandling) << "RooWorkSpace::import(" << GetName() << ") ERROR object named " << node->GetName()
 			        << " has an error in importing in one or more of its auxiliary objects, aborting" << endl ;
       return kTRUE ;
     }
   }
-  iter->Reset() ;
-
-  if (cloneSet2->getSize()+_allOwnedNodes.getSize() > 999) _allOwnedNodes.setHashTableSize(1000);
```

Comment:
was the removal of this line intentional? if yes, should it maybe be clarified in the commit message or even be moved to its own commit?

Reply:
Intentional, yes. SetHashTableSize wasn't doing anything, but could split the commit.